deployment/k8s/charts/aggregates/migrationAssistant/values.yaml

```diff
@@ -103,8 +109,18 @@ bulk-document-loader:
       source: otherConfig
       configMapName: "target-cluster"
       configMapKey: "basicAuthPassword"
+    sourceVersion:
+      source: parameterConfig
+      value: "ES_7.10"
+      allowRuntimeOverride: true
+    docTransformerConfigFile:
+      source: parameterConfig
+      value: "/shared-logs-output/test-transformations/transformation.json"
```

Comment:
See https://opensearch.atlassian.net/browse/MIGRATIONS-2401 for a better way to send configs/arguments into processes.